hscontrol/policy/v2/policy.go

```diff
@@ -664,41 +664,41 @@ func (pm *PolicyManager) invalidateAutogroupSelfCache(oldNodes, newNodes views.S
 	// Check for removed nodes
 	for nodeID, oldNode := range oldNodeMap {
 		if _, exists := newNodeMap[nodeID]; !exists {
-			affectedUsers[oldNode.User().ID] = struct{}{}
+			affectedUsers[oldNode.User().Model().ID] = struct{}{}
```

Comment:
On the UserView, we should have a ID() func that maps .Model().ID so we do not have to write it out in full everywhere. do it and replace all the `.Model().ID` calls.